src/forms/TimeInput.jsx

```diff
@@ -69,12 +69,13 @@ class TimeInput extends React.Component {
 
 	/**
 	* @function onChange
-	* @param partialState the information to update state with
+	* @param {Object} e the information to update state with
```

Comment:
Doc comment just looks like a copy/paste from the next line... but why do we need to pass in this event & through to the callback? Where's the callback needed?

Reply:
Same question.

Reply:
The callback is used in RsvpTimes in `mup-web`.
RsvpTimes flips another value (`rsvp_times_active`) when an onChange occurs.
It needs other info from the event, not just the value.

Reply:
actually i may have a way to not need the event passed to the callback... but do you think its harmful?

Reply:
IMO anythin called `onChange` should _always_ be given an `Event` argument, regardless of whether it's used, to stay consistent with native onChange event handler behavior

Reply:
Definitely don't think it's harmful (and will echo @mmcgahan's sentiment), just wanted to know where it was coming from!